nexus/db-model/src/dataset.rs

```diff
@@ -41,6 +41,7 @@ pub struct Dataset {
 
     pub kind: DatasetKind,
     pub size_used: Option<i64>,
+    zone_name: Option<String>,
```

Comment:
Should this be `pub`? Or does it need a getter?

Reply:
I'd rather keep it not-pub, but it's used in a follow-up PR (https://github.com/oxidecomputer/omicron/pull/6229) when converting this type into a format that can be sent to `datasets PUT` on sled agents